### PR TITLE
Fix #871

### LINKED ISSUE #871: capture group not supported; use '(?:' instead of '('
**Describe the bug**

```
[rdlugyhe@rdlugyhe-thinkpadt14sgen2i rhbq-quarkus]$ vale sync
 SUCCESS  Downloaded package 'RedHat'                                                                                                                                                         
 SUCCESS  Downloaded package 'AsciiDoc'                                                                                                                                                       
Downloading packages [2/2] █████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████ 100% | 1s
[rdlugyhe@rdlugyhe-thinkpadt14sgen2i rhbq-quarkus]$ vale modules/proc_updating-quarkus.adoc
E201 Invalid value [/home/rdlugyhe/gitlab/rhbq-quarkus/.vale/styles/RedHat/TermsErrors.yml:22:4]:

  21    "(?<!self-)hosted engine|hosted-engine": self-hosted engine
  22*   "bare metal( clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| instance?| machines?| media| nodes?| provisioning| servers?| workers?)": bare-metal$1
  23    "bare-metal(?! clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| instance?| machines?| media| nodes?| provisioning| servers?| workers?)": bare metal
  24    "[bB]asic [aA]uth": Basic HTTP authentication|Basic authentication

capture group not supported; use '(?:' instead of '('

Execution stopped with code 1.
[rdlugyhe@rdlugyhe-thinkpadt14sgen2i rhbq-quarkus]$
```
Examples:

* This word or expression triggers an illegitimate vale alert.
* This word or expression triggers an ambiguous vale alert.

**To Reproduce**

1. Add this word or expression:
2. Vale reports an illegitimate alert: error, warning, suggestion.

**Expected behavior**

**Additional context**
Add any other context here.

### PR DESCRIPTION
**Pull request by rolfedh**
Fix https://github.com/redhat-documentation/vale-at-red-hat/issues/871

### CODE REVIEW

**Comment by aireilly** [comment]
This doesn't work like you think it does @rolfedh 
image

This rule uses a new feature - capture group support ($1) see https://github.com/errata-ai/vale/releases/tag/v3.7.0

File: .vale/styles/RedHat/TermsErrors.yml [hunk 1]
```diff
@@ -19,8 +19,8 @@ swap:
   "(?<!Mozilla )Thunderbird": Mozilla Thunderbird
   "(?<!pseudo-)ops": operations
   "(?<!self-)hosted engine|hosted-engine": self-hosted engine
-  "bare metal( clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| instance?| machines?| media| nodes?| provisioning| servers?| workers?)": bare-metal$1
-  "bare-metal(?! clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| instance?| machines?| media| nodes?| provisioning| servers?| workers?)": bare metal
+  "bare metal(?: clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| instances?| machines?| media| nodes?| provisioning| servers?| workers?)": bare-metal$1
```
Comment (aireilly) [inline .vale/styles/RedHat/TermsErrors.yml:22]:
```suggestion
  "bare metal( clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| instances?| machines?| media| nodes?| provisioning| servers?| workers?)": bare-metal$1
```

**Review by apinnick — COMMENTED** [review]
I do not think I can provide a meaningful review. I am too new to this project and not sure how to test the proposed change.

**Review by apinnick — APPROVED** [review]
I agree with the "instances" change. 
Not sure about Aidan's suggestion.

**Comment by rolfedh** [comment]
I've updated from Vale version 2.x to 3.x. Aidan updated the docs to reflect this version requirement. Updating this PR to reflect Aidan's suggestion.

**Comment by rolfedh** [comment]
The current PR simply replaces "instance" with "instances".

**Review by aireilly — APPROVED** [review]
/lgtm

**Pull request merged**